OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -63,6 +63,15 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Arc in WAngles, two values indicate variable arc.")]
 		public readonly WAngle[] LaunchAngle = { WAngle.Zero };
 
+		[Desc("Up to how many times does this bullet bounce when touching ground without hitting a target?")]
+		public readonly int BounceCount = 0;
```

Comment:
I would change this to `MaxBounceCount` and the desc to something like `If 0 this projectile will be destroyed after N ticks / after M distance`.